cg/apps/lims/api.py

```diff
@@ -244,24 +290,28 @@ def get_prep_method(self, lims_id: str) -> str:
 
     def get_sequencing_method(self, lims_id: str) -> str:
         """Get the sequencing method."""
-        process_names = [
-            'CG002 - Cluster Generation (HiSeq X)',
-            'CG002 - Cluster Generation (Illumina SBS)',
-        ]
+
```

Comment:
I get the feeling from looking at this method that is really similar to the one above, can they be made into one?

Reply:
awesome!